contracts/interchain-token/src/storage_types.rs

```diff
@@ -13,6 +13,7 @@ pub struct AllowanceValue {
     pub expiration_ledger: u32,
 }
 
+/// Do not use `METADATA_KEY`` as it is reserved for token metadata.
```

Comment:
```suggestion
/// Do not use the symbol `METADATA` as a key as it is reserved for token metadata.
```
we're interested in knowing the actual value, and not the variable name they use